### PR TITLE
adds a confirmation prompt for blob and flock deployment

### PR DESCRIPTION
**Pull request by nefarious6th**
[gamemodes][qol]

## About the PR 
adds a confirmation prompt to blobs deploying their first nucleus and flocks spawning their rift.

## Why's this needed? 
Verification
preventing blobs and flocks from misclicking and deploying

### CODE REVIEW

**Comment by TobleroneSwordfish** [comment]
Oh and if you're adding it for blob you may as well add it for flock too for consistency

File: code/modules/antagonists/blob/blob_abilities.dm [hunk 1]
```diff
@@ -191,6 +191,9 @@
 
 	onUse(var/turf/T)
 		if (..())
```
[the remark this answers is not in the document]
Comment (nefarious6th) [reply code/modules/antagonists/blob/blob_abilities.dm:193]: just remove the `TRUE`?

[the remark this answers is not in the document]
Comment (nefarious6th) [reply code/modules/antagonists/blob/blob_abilities.dm:193]: oh, whoops, i get it now, i moved the section under all the stuff you can't deploy into

File: code/datums/abilities/flock/flockmind.dm [hunk 2]
```diff
@@ -92,6 +92,10 @@
 
 	var/turf/T = get_turf(F)
 
+	if (F)
+		if (tgui_alert(F,"Would you like to spawn a rift?","Spawn Rift?",list("Yes","No")) != "Yes")
+			return TRUE
+
```
Comment (TobleroneSwordfish) [inline code/datums/abilities/flock/flockmind.dm:98]: I'd move this below the area checks so you can use it to see if the area you've selected is valid without actually deploying.

File: code/modules/antagonists/blob/blob_abilities.dm [hunk 3]
```diff
@@ -227,6 +227,10 @@
 						boutput(owner, "<span class='alert'>You are being watched.</span>")
 						return
 
+		if (owner)
+			if (tgui_alert(owner,"Would you like to deploy your nucleus?","Deploy Nucleus?",list("Yes","No")) != "Yes")
+				return TRUE
+
```
Comment (TobleroneSwordfish) [inline code/modules/antagonists/blob/blob_abilities.dm:233]: Again, put it below the tutorial check so you don't get a confirmation prompt if you can't use the ability

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]

**Pull request merged**